app/cli.php

```diff
@@ -99,6 +102,48 @@
     return $dbForPlatform;
 }, ['pools', 'cache']);
 
+CLI::setResource('console', function () {
```

Comment:
Eldad suggested moving the console resource to an associative array inside https://github.com/appwrite/appwrite/tree/1.6.x/app/config/console.php and reusing that array across init.php ,cli.php , in workers and in other places in the future.